### ISSUE TITLE
Why is style and javascript seperated

### ISSUE THREAD

**Issue by sp90** [lukeed/preact-starter#13]
Hi again, 

I just want to clarify why is scss/sass separated from javascripts, in components - is there a reason?

**Comment by lukeed** [comment]
Hey, it's just my own preference. Before css modules/CSS-in-JS came along, this was the norm. I'd venture to say that in most circles, this still _is_ the norm.

**Comment by sp90** [comment]
No doubt, but it is a component based setup, which anyhow would make sense so you could grab a component folder at put it somewhere else right?

**Comment by lukeed** [comment]
Yes, but your styles structure would (generally) match your views structure. 

Just another way of doing things. Has no issue scaling in my experience. =]

The `preact-boilerplate` is nearly identical to this starter, with the exception of its css modules. Maybe that's what you're after?

**Comment by sp90** [comment]
Nope there where greater losses - im just trying to understand your choices 👍

**Comment by lukeed** [comment]
I might write a blog post on this 😲😛 I'm really against the practice in general.

// cc @zouhir
